cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/task/interactivelogin/AzureRoleManager.java

```diff
@@ -247,21 +248,17 @@ private boolean validateRoleActions(AzureRoleDefinition role) {
             actions.addAll(permission.getActions());
             notActions.addAll(permission.getNotActions());
         }
-        if (actions.contains("*")) {
-            return !notActions.contains("Microsoft.Compute/*")
-                    && !notActions.contains("Microsoft.Authorization/*/read")
-                    && !notActions.contains("Microsoft.DataLakeStore/accounts/read")
-                    && !notActions.contains("Microsoft.Network/*")
-                    && !notActions.contains("Microsoft.Storage/*")
-                    && !notActions.contains("Microsoft.Resources/*");
-        } else {
-            return actions.contains("Microsoft.Compute/*")
-                    && actions.contains("Microsoft.Authorization/*/read")
-                    && actions.contains("Microsoft.DataLakeStore/accounts/read")
-                    && actions.contains("Microsoft.Network/*")
-                    && actions.contains("Microsoft.Storage/*")
-                    && actions.contains("Microsoft.Resources/*");
-        }
+        return actions.contains("*") ? !notActions.contains("Microsoft.Compute/*")
```

Comment:
the semicolon should be in a new line, this format looks like a single expression

Reply:
You are right